cmd/discovery.go

```diff
@@ -21,6 +21,9 @@ type esnode struct {
 	version string
 }
 
+const ELASTICSEARCH_SERVICE = "elasticsearch-all"
```

Comment:
It is criteo specific, so it should be removed or the project not be open-source